ProGet/installation/installation-guide/traditional-installer.md

```diff
@@ -1,108 +1,175 @@
-
-## 8. Target Directory {#target data-title="Target Directory"}
-
-Deploy to your favorite location, or just stick with the default of Program Files.
-
-![](/resources/documentation/proget-installation/progettargetdirectory.png){.screenshot}
-
-## 9. Database Configuration {#configuration data-title="Database Configuration"}
-
-If you ran the installer with the embedded SQL Express, then you'll be able to select the option to install SQL Express:
-
-![](/resources/documentation/proget-installation/progetdatabase.png){.screenshot}
-
-Otherwise, you'll need to point to an existing server using a standard SQL Connection string. Note that if no database is specified in the connection string (which is the default), a database named ProGet will be created.
-
-## 10. Web Server Configuration {#web-config data-title= "10. Web Server Configuration"}
-
-By default, the installer will configure ProGet to use its own web server. This is the easiest to start with, and you can always switch to IIS hosting later.
-
-![](/resources/documentation/proget-installation/progetwebserver.png){.screenshot}
-
-If IIS is installed on the server, you will have the option of installing ProGet in IIS. This will create an application pool and website in the local IIS configuration.
-
-## 11. User Account {#user-account data-title="11. User Account"}
-
-By default, the installer will use the Network Service account to run the ProGet Service and Web Application. We Recommend sticking with this, and changing the account later if you need to.
-
-![](/resources/documentation/proget-installation/useraccount.png){.screenshot}
-
-## 12. Summary {#summary data-title="12. Summary"}
-
-Make sure everything here looks OK, and then click Install.
-
-![](/resources/documentation/proget-installation/summary.png){.screenshot}
-
-If any of the items indicate an error, you can click on it to go back to that page and fix it.
-
-## 13. Patiently Wait {#patiently-wait data-title="Patiently Wait"}
-
-If you chose to have SQL Server Express installed, then the actual installation process will take a while, as the installer must download the SQL installer and then wait for it to complete. On some servers, we've seen this take as long as fifteen minutes.
-
-![](/resources/documentation/proget-installation/progetclickinstall.png){.screenshot}
-
-Clicking cancel (especially during the SQL Express installation) is not advised, as you may have to manually remove some bits of SQL Server using Add/Remove programs. Once everything's done, you'll be ready to start using ProGet.
+*We are planning to stop shipping legacy installer in 2021*
+
+For the most convenient way to install ProGet is to use the [InedoHub](https://my.inedo.com/downloads). It is primarily used to install, update, and otherwise service applications in the Inedo Suit. You can learn more about it on how the InedoHub can help you save time by reading our [guide](https://docs.inedo.com/docs/desktophub/overview).
```

Comment:
* first link to Inedo Hub should be the overview docs (i.e. ttps://docs.inedo.com/docs/desktophub/overview)
* fix typos
* Inedo Hub (has a space)
* reading our [guide] --> [reading our guide]